src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationHandler.cs

```diff
@@ -217,6 +218,47 @@ public OpenIdConnectAuthenticationHandler(HttpClient backchannel)
 
             message.State = Options.StateDataFormat.Protect(properties);
 
+            if (Options.UsePostAuthorizationRequests)
+            {
+                using (var buffer = new MemoryStream())
+                using (var writer = new StreamWriter(buffer))
+                {
+                    writer.WriteLine("<!doctype html>");
```

Comment:
Yep looks good, I don't have enough context about the actual feature so that's about all I can contribute to this PR :)